x-pack/plugins/index_management/public/application/lib/indices.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import SemVer from 'semver/classes/semver';
+import { Index } from '../../../common';
+
+const version = '8.0.0';
```

Comment:
In `7.x` this will be `7.16.0`

Reply:
Does this need to be hardcoded? I think you can read the Kibana version from core via `PluginInitializerContext.env.packageInfo.version`. See https://github.com/elastic/kibana/issues/85969#issuecomment-745372457.

Reply:
I agree with Alison. Would we have to remember to update these strings for each patch release (`7.16.1`, `8.0.1`) and minor (`8.1.0`)?

Reply:
I started trying to inject dynamically the version with `PluginInitializerContext.env.packageInfo.version`. Then I realised it was going to change way too many files for very little benefit. I then weighted the pros and cons and thought it was much more surgical with less impact in the code base (less risk of regression) to have 2 hardcoded strings.

> Would we have to remember to update these strings for each patch release

No we won't, the code only looks for the major version.